src/modules/reading-room/components/FilterMenu/FilterMenu.module.scss

```diff
@@ -91,13 +97,29 @@
 		z-index: get-z-layer("filter");
 	}
 
+	:global(.c-menu) {
+		padding: $c-filter-menu-flyout-top-padding 0 0;
+	}
+
+
 	&--filter {
+		max-width: 46.4rem;
+
 		// Overwrite popper styles
 		min-width: 46.4rem !important;
 		max-height: 100vh;
 	}
 }
 
+.c-filter-menu__flyout-close {
+	position: absolute;
+	inset: 1.2rem 1.2rem auto auto;
+}
+
 .c-filter-menu__overlay {
 	z-index: get-z-layer("filter-overlay");
 }
+
+.c-filter-menu__form {
+	max-height: calc(100vh - $c-filter-menu-flyout-top-padding);
+}
```

Comment:
Op basis van https://xd.adobe.com/view/86b0be3f-d968-4a98-7784-d5d00cd42f98-f081/screen/0abdd661-dc35-4509-a484-76c882ed462a/specs/ zou ik zeggen dat de `reset` & `pas toe` buttons steeds above-the-fold moeten blijven. In kader daarvan lijkt me een flex layout hier beter, maar complexer, als een max-height

Reply:
Dit is het geval, pas als de max-height bereikt is gaat de body scrollable worden maar blijven de form acties steeds zichtbaar. De flex layout wordt toegepast in https://github.com/viaacode/hetarchief-client/pull/94/files#diff-dc2577963573e5ecefd03a9721af3908a5717f8c04c0b961857353c3738851c3